docs/body.md

```diff
@@ -295,6 +295,10 @@ The following daemons are installed:
 - zOpenStackHostMapSame: A list of \<name1\>=\<name2\>,
   used to inform the modeler that the same host may be referred to with an
   alternate name by some part of OpenStack. (this is not commonly used)
+- zOpenStackHostMapPrefix: Prefix to use when adding OpenStack Host as a Linux Device. Primary use-case,
```

Comment:
Since this only affects the device's ID, not the component's, i'd probably suggest zOpenStackHostDevicePrefix or something along those lines

Reply:
Updated zProperty to new name.